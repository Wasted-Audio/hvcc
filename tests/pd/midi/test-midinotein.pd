#N canvas 572 166 503 302 12;
#X obj 97 61 notein;
#X obj 97 132 print;
#X obj 97 96 pack f f f;
#X connect 0 0 2 0;
#X connect 0 1 2 1;
#X connect 0 2 2 2;
#X connect 2 0 1 0;
