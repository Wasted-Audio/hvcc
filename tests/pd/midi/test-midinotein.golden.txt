[@ 0.000] print: 48 64 2
[@ 0.000] print: 72 64 1
[@ 0.000] print: 48 0 2
[@ 0.000] print: 72 0 1
[@ 0.000] print: 60 64 2
[@ 0.000] print: 72 64 1
[@ 0.000] print: 60 0 2
[@ 0.000] print: 72 0 1
[@ 0.000] print: 64 64 2
[@ 0.000] print: 79 64 1
[@ 0.000] print: 64 0 2
[@ 0.000] print: 79 0 1
[@ 0.000] print: 60 64 2
[@ 0.000] print: 79 64 1
[@ 0.000] print: 60 0 2
[@ 0.000] print: 79 0 1
[@ 0.000] print: 65 64 2
[@ 0.000] print: 81 64 1
[@ 0.000] print: 65 0 2
[@ 0.000] print: 81 0 1
[@ 0.000] print: 60 64 2
[@ 0.000] print: 81 64 1
[@ 0.000] print: 60 0 2
[@ 0.000] print: 81 0 1
[@ 0.000] print: 64 64 2
[@ 0.000] print: 79 64 1
[@ 0.000] print: 64 0 2
[@ 0.000] print: 60 64 2
[@ 0.000] print: 60 0 2
[@ 0.000] print: 79 0 1
[@ 0.000] print: 62 64 2
[@ 0.000] print: 77 64 1
[@ 0.000] print: 62 0 2
[@ 0.000] print: 77 0 1
[@ 0.000] print: 59 64 2
[@ 0.000] print: 77 64 1
[@ 0.000] print: 59 0 2
[@ 0.000] print: 77 0 1
[@ 0.000] print: 60 64 2
[@ 0.000] print: 76 64 1
[@ 0.000] print: 60 0 2
[@ 0.000] print: 76 0 1
[@ 0.000] print: 57 64 2
[@ 0.000] print: 76 64 1
[@ 0.000] print: 57 0 2
[@ 0.000] print: 76 0 1
[@ 0.000] print: 53 64 2
[@ 0.000] print: 74 64 1
[@ 0.000] print: 53 0 2
[@ 0.000] print: 74 0 1
[@ 0.000] print: 55 64 2
[@ 0.000] print: 74 64 1
[@ 0.000] print: 55 0 2
[@ 0.000] print: 74 0 1
[@ 0.000] print: 48 64 2
[@ 0.000] print: 72 64 1
[@ 0.000] print: 48 0 2
[@ 0.000] print: 72 0 1